src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngine.java

```diff
@@ -384,11 +389,20 @@ private GenotypesContext annotateGenotypes(final ReferenceContext ref,
         final GenotypesContext genotypes = GenotypesContext.create(vc.getNSamples());
         for ( final Genotype genotype : vc.getGenotypes() ) {
             final GenotypeBuilder gb = new GenotypeBuilder(genotype);
-            genotypeAnnotations.stream().filter(addAnnot).forEach(annot -> annot.annotate(ref, vc, genotype, gb, likelihoods));
+            for ( final GenotypeAnnotation annotation : genotypeAnnotations) {
 
-            if (fragmentLikelihoods.isPresent() && haplotypeLikelihoods.isPresent()) {
-                jumboGenotypeAnnotations.stream().filter(addAnnot).forEach(annot ->
-                        annot.annotate(ref, features, vc, genotype, gb, likelihoods, fragmentLikelihoods.get(), haplotypeLikelihoods.get()));
+                genotypeAnnotations.stream().filter(addAnnot)
+                        .forEach(annot -> annot.annotate(ref, vc, genotype, gb, likelihoods));
```

Comment:
ilya - check - maybe reassign

Reply:
@ilyasoifer I spoke with @jamesemery about this one, and he confirmed that it's almost certainly a bug. This could actually have a noticeable impact on performance since it's turning a O(n) operation on the genotypes into a O(n^2) operation.

Reply:
@jamesemery So it sounds like you are handling this one?

Reply:
This looks like rebase gore to me. If you look its also kindly re-adding the genotype annotations twice in a row... and its also repeating this whole process N times... whoops...